R/effectsize.R

```diff
@@ -153,20 +202,21 @@ effectsize.BFBayesFactor <- function(model, ...){
       posts <- posts * N
     }
 
-    V <- apply(posts, 1, function(a) {
-      cramers_v(matrix(a, nrow = nrow(data)), ci = NULL)[[1]]
+    ES <- apply(posts, 1, function(a) {
+      f(matrix(a, nrow = nrow(data)), ci = NULL)[[1]]
     })
 
-    res <- data.frame(Cramers_v = V)
+    res <- data.frame(ES)
+    colnames(res) <- colnames(f(matrix(posts[1,], nrow = nrow(data)), ci = NULL))
   } else if (inherits(model@numerator[[1]], c("BFoneSample", "BFindepSample"))) {
     D <- as.matrix(BayesFactor::posterior(model, iterations = 4000, progress = FALSE))[,"delta"]
     res <- data.frame(Cohens_d = D)
   } else if (inherits(model@numerator[[1]], "BFcorrelation")) {
     rho <- insight::get_parameters(model)[["rho"]]
     res <- data.frame(r = rho)
```

Comment:
To be internally consistent, I don't think this should be renamed to `r`:

``` r
library(BayesFactor)
library(easystats)

bf <- correlationBF(y = iris$Sepal.Length, x = iris$Sepal.Width)

effectsize::effectsize(bf)
#> # Description of Posterior Distributions
#> 
#> Parameter | Median |          89% CI |     pd |        89% ROPE | % in ROPE
#> ---------------------------------------------------------------------------
#> r         | -0.113 | [-0.237, 0.009] | 92.70% | [-0.100, 0.100] |    41.309

parameters::model_parameters(bf)
#> Parameter | Median |        89% CI |     pd | % in ROPE |              Prior |    BF
#> ------------------------------------------------------------------------------------
#> rho       |  -0.11 | [-0.24, 0.01] | 92.10% |    41.95% | Cauchy (0 +- 0.33) | 0.509
```

<sup>Created on 2020-12-03 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0.9001)</sup>